### ISSUE TITLE
`circle-animation-progress` event doesn't trigger if the circle is rendered in background

### ISSUE THREAD

**Issue by pooriajahedi** [kottenator/jquery-circle-progress#166]
But after page refresh, everything is correct.
I'm using wordpress. This is my website: www.fenix-team.com.
What should I do? Thanks!

**Comment by kottenator** [comment]
I don't see where is the circle on your page. Please, provide precise steps to reproduce the problem.

**Comment by pooriajahedi** [comment]
Please, open this link: https://www.fenix-team.com.
Hover these boxes and you will see it:
![image](https://user-images.githubusercontent.com/6542099/30000814-236f21b2-908e-11e7-8fba-ebf6d81c2702.png)
Right click and select "open link in new page".
Wait until page is loaded. Then go to that page opened.
![image](https://user-images.githubusercontent.com/6542099/30000882-e1f0f316-9090-11e7-88a4-040eade1e9b8.png)
The value inside the circle is not loaded.

**Comment by pooriajahedi** [comment]
@kottenator  hi.
I've found the same bug on your plugin page. Before the page is completely loaded, go to another page or minimize that page. And after it's completely loaded, open the page - the value inside the progress bar is not rendered.

![image](https://user-images.githubusercontent.com/6542099/30014096-99d440d6-915f-11e7-958c-946f319dae6c.png)

**Comment by kottenator** [comment]
Thanks, @pooria-jahedi. I can reproduce the bug and I'll try to fix it.

**Comment by pooriajahedi** [comment]
@kottenator Hi sir, 
Did you fix it ?

**Comment by kottenator** [comment]
Not yet - pretty busy at work, almost no free time :( But I'll come to this bug sooner or later. If it's urgent for you - you could try to fix it and make a PR.

**Comment by pooriajahedi** [comment]
@kottenator Hi sir,
Did you fix it ?

**Comment by kottenator** [comment]
No, sorry, I didn't have a chance to look at this problem yet. I'd appreciate your help, if you have time to investigate this.
